dll/win32/shell32/CDefView.cpp

```diff
@@ -1453,9 +1504,9 @@ HRESULT CDefView::FillList(BOOL IsRefreshCommand)
 
     /* sort the array */
     int sortCol = -1;
-    if (!IsRefreshCommand) // Are we loading for the first time?
+    if (!IsRefreshCommand && !m_sortInfo.bLoadedFromViewState) // Are we loading for the first time?
     {
-        m_sortInfo.bIsAscending = TRUE;
+        m_sortInfo.Direction = 0;
```

Comment:
What's the meaning of Direction == 0 ? (Maybe this would provide me with some extra info regarding my previous question)

Reply:
You can think of it as undefined with a slight lean towards sort ascending. If nobody outside DefView has any opinion on the matter either, the sort function in your previous question just has to pick a direction and it will pick ascending.